# Generated with epl_ros_generator.py

# openPOWERLINK variables:
string shm_name
uint16 pid

# xap.h variables:
uint8 CN1_DigitalOutput_00h_AU8_DigitalOutput
uint8 CN32_DigitalOutput_00h_AU8_DigitalOutput
uint8 CN110_DigitalOutput_00h_AU8_DigitalOutput
